### PR TITLE
Two new posters

### PR DESCRIPTION
**Pull request by TillsterRulz**
## About the PR
Added two news posters for the server. Art done by me but the welding mask and microwave are identical to those ingame.

## Why / Balance
More posters is more fun

## Technical details
Added The Maints Slasher poster and the PSA: DO NOT GAMBLE poster with befitting descriptions.

## Media
other PR for posters didnt have pictures so it should be fine right haha? they are in the files as 32x32 pngs

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Should be none, I pretty much just copy pasted code

**Changelog**
:cl:
- added: The Maints Slasher poster. Just who was this "slasher" and where are they now?
- added: PSA: DO NOT GAMBLE poster. Dont get caught microwaving your ID when you see this poster!

### CODE REVIEW

**Comment by TillsterRulz** [comment]
forgot to actually add the posters in to the spawner menu, so i went back and changed it

File: Resources/Textures/_Harmony/Structures/Wallmounts/posters.rsi/meta.json
Comment (FluffMe) [inline Resources/Textures/_Harmony/Structures/Wallmounts/posters.rsi/meta.json:1]: Please add yourself to the "copyright" field.

**Review by FluffMe — CHANGES_REQUESTED** [review]
Missing copyright.
Otherwise, looks good to me.

**Comment by FluffMe** [comment]
> other PR for posters didnt have pictures so it should be fine right haha? they are in the files as 32x32 pngs
We usually add in-game screenshots of the posters for reviewers to make a decision on the addition.

image
image

I'll leave final decision to Keld.

@KeldWolf, if you merge, please squash

**Review by KeldWolf — APPROVED** [review]
Love seeing more of these.

**Comment by KeldWolf** [comment]
I also like that they're reasonably sized.

**Comment by FluffMe** [comment]
@KeldWolf you forgot the squash! :'(

**Pull request merged**